templates/templated_email/account/account_delete.email

```diff
@@ -5,9 +5,8 @@
 {% endblock %}
 
 {% block plain %}
+{% trans "Hi!" context "Standard e-mail greeting" %}
 {% blocktrans context "Account delete reset e-mail text" %}
```

Comment:
Hi,

Since the changes are breaking the translations, could you make sure that all blocktrans in emails templates are set to be `trimmed`? That would be awesome!